master/internal/kubernetes/workers.go

```diff
@@ -0,0 +1,97 @@
+package kubernetes
```

Comment:
nonblocking: probably rename this file to be `request_worker.go` or something like that.

Reply:
done, renamed to `request_workers.go`